elasticapm/instrumentation/packages/base.py

```diff
@@ -131,6 +172,15 @@ def uninstrument(self):
         self.originals = {}
 
     def call_if_sampling(self, module, method, wrapped, instance, args, kwargs):
+        """
+        This is the function which will wrap the instrumented method/function.
+
+        By default, will call the instrumented method/function, via `call()`,
+        only if a transaction is active. This behavior can be overridden by
```

Comment:
Might be worth pointing out here what happens if a transaction is active but `is_sampled` set to `False`.